src/channel_state.ts

```diff
@@ -546,19 +639,6 @@ export class ChannelState<
         return messageArr.set(left - 1, message);
     }
 
-    const mutable = messageArr.asMutable() as Array<
-      ReturnType<
-        ChannelState<
-          AttachmentType,
-          ChannelType,
-          CommandType,
-          EventType,
-          MessageType,
-          ReactionType,
-          UserType
-        >['messageToImmutable']
-      >
-    >;
     mutable.splice(left, 0, message);
```

Comment:
looks like an accidental delete

Reply:
I moved it to the top of the method because I need this type for `sortBy`